bundle/bundle_test.go

```diff
@@ -246,3 +247,41 @@ func TestReadCustomExtensions(t *testing.T) {
 	assert.Equal(t, true, backupExt["enabled"])
 	assert.Equal(t, "daily", backupExt["frequency"])
 }
+
+func TestBundle_RoundTripEmptyJson(t *testing.T) {
+	wantData, err := ioutil.ReadFile("testdata/empty.json")
```

Comment:
You can merge those two tests and use the [TableDrivenTesting](https://github.com/golang/go/wiki/TableDrivenTests) pattern here, with something like this:

```go
func TestBundle_RoundTrip(t *testing.T){
    testCases := []struct{
        name string
        json string
    }{
        { name: "EmptyJson", json: "testdata/empty.json"},
        { name: "MinimalJson", json: "testdata/minimal.json"},
    }
    for _, tc := range testCases {
        t.Run(tc.name, func(t *testing.T) {
            wantData, err := ioutil.ReadFile(tc.json)
            ...
        }
    }
}
```